src/layers.rs

```diff
@@ -306,8 +336,12 @@ use crate::cfg::Cfg;
 
 #[test]
 fn test_mgr() {
-    let cfg = Cfg::new(vec![
-        // 0: base layer
+    let mut h = HashMap::new();
```

Comment:
We can add a few test cases while we're at it.
E.g one "happy" path test for a valid alias toggle, and one for a bad alias (shouldn't affect the layer state)

Reply:
Sounds great, good call